tests/bwc/test_rolling_upgrade.py

```diff
@@ -14,7 +13,7 @@ def __repr__(self):
 
 
 ROLLING_UPGRADES = (
```

Comment:
should we add also: `UpgradePath('3.1.x', 'latest-nightly')` ?

Reply:
Can't add it yet as we've no `3.1` releases yet

Reply:
yeap sorry, stupid me

Reply:
I've had the same thought first :)